docs/bash_to_xsh.rst

```diff
@@ -68,6 +68,9 @@ line is ``#!/usr/bin/env xonsh``.
     * - ``$@``
       - ``$ARGS``
       - List of all command line argument and parameter strings.
+    * - ``exit``
+      - ``raise SystemExit`` or ``sys.exit()``
```

Comment:
While `raise SystemExit` does do it, I've literally never seen this in python code. I would not encourage its use.

Reply:
304k in GitHub search - https://github.com/search?q=%22raise+SystemExit%22&type=Code

Reply:
That doesn't mean it's a good idea or idiomatic or something we should actively encourage.

Reply:
The best way is to have just `exit()` function by default without any imports. Ok, I remove SystemExit.